client/test/cypress/integration/submissiondetails.spec.js

```diff
@@ -69,20 +53,13 @@ describe("Submissions Details", () => {
     cy.dataCy("review_coordinator_assignee_selected").contains(
       "applicationAdminUser"
     )
-    cy.dataCy("input_review_coordinator_assignee").type(
-      "test{backspace}{backspace}{backspace}{backspace}"
-    )
     cy.dataCy("button_assign_review_coordinator").click()
     cy.dataCy("list_assigned_review_coordinators").contains(
       "Application Administrator"
     )
     cy.injectAxe()
     cy.dataCy("submission_details_notify")
-    cy.checkA11y(null, {
-      rules: {
-        "nested-interactive": { enabled: false },
-      },
-    }, a11yLogViolations)
+    cy.checkA11y(null, null, a11yLogViolations)
```

Comment:
The removal of the exceptions to ignore the "nested-interactive" rule was the only questionable thing I noticed. Has this been resolved? If so, there are a couple other Cypress specs that include this rule exception that will also need to be updated eventually; this doesn't have to be done in this PR.

Reply:
Removed the rest of the nested interactive options, but there are still underlying a11y issues with QSelect.